diesel/src/upsert/on_conflict_target.rs

```diff
@@ -1,9 +1,10 @@
+use backend::{Backend, SupportsOnConflictClause};
 use expression::SqlLiteral;
-use pg::Pg;
 use query_builder::*;
 use query_source::Column;
 use result::QueryResult;
 
+#[cfg(feature = "postgres")]
```

Comment:
Move postgres specific implementations back to the `pg/upsert` module and then re-export them here.

Reply:
We should do `pub pg::on_constraint::*;` to have the old module layout in place. Otherwise this would be a breaking change.